Feat/issue 175 brokenannexeslinks

skounis opened the pull request:
Resolves #175

pinamiranda reviewed (changes requested):
Change in line 207 needs to be reviewed. The link is to "annexes/annex-2/annex_2-high-level-requirements.md" and it should be to  "annexes/annex-2/annex-2-high-level-requirements.md"

skounis commented:
> Change in line 207 needs to be reviewed. The link is to "annexes/annex-2/annex_2-high-level-requirements.md" and it should be to "annexes/annex-2/annex-2-high-level-requirements.md"

@pinamiranda Thank you for the attention. I fixed it and updated the PR
* https://github.com/eu-digital-identity-wallet/eudi-doc-architecture-and-reference-framework/pull/183/commits/28532af728e21c875f720c933995b4202aa10ef4
* https://github.com/eu-digital-identity-wallet/eudi-doc-architecture-and-reference-framework/pull/183/files#diff-42bd4374f09a239a1feca1e6f45b7aa69e7a59b96d7cbc327414d1109ba59b3fR207
*

pinamiranda reviewed (approved):
Agree
